### ISSUE TITLE
Can't import iframe-resizer with webpack

### ISSUE THREAD

**Issue by anlambert** [davidjbradshaw/iframe-resizer#674]
When I try to import the `iframe-resizer` module into my webpack project, using syntax `require('iframe-resizer)` or `import iframe-resizer`, I got the following error:
```
ERROR in ./node_modules/iframe-resizer/index.js
Module not found: Error: Can't resolve './js' in '/home/antoine/swh/swh-environment/swh-web/node_modules/iframe-resizer'
 @ ./node_modules/iframe-resizer/index.js 3:17-32
 @ ./swh/web/assets/src/bundles/vendors/index.js
 @ multi bundles/vendors/index.js
```

My guess is that the index file inside the `js` folder is named `Index.js` but it should be named `index.js` (no uppercase characters).

**Comment by davidjbradshaw** [comment]
Argh, will try and take a look this evening

**Comment by davidjbradshaw** [comment]
If you have a moment to make a PR, that will help, my day job blocks access to github :(

**Comment by anlambert** [comment]
PR #675 opened fixing the issue (I have tested locally and the import error goes away).

**Comment by davidjbradshaw** [comment]
Cool I will publish this tonight when I get home, thanks for the help

**Comment by gavmck** [comment]
Literally came across this issue 5 minutes ago, what timing!

**Comment by davidjbradshaw** [comment]
Published as v4.0.4, sorry for the silly mistake and thanks for the help

**Comment by dev-kaan** [comment]
Problem persists in angular
